typescript/nx.json

```diff
@@ -8,5 +8,10 @@
                 "cacheableOperations": ["build", "lint", "test", "e2e"]
             }
         }
+    },
+    "targetDefaults": {
```

Comment:
Does this enforce build dependency (and replace the nx.targets.build... rule in well-log-viewer/package.json ?